hail/python/hailtop/utils/utils.py

```diff
@@ -220,6 +220,8 @@ def is_transient_error(e):
         return True
     if isinstance(e, asyncio.TimeoutError):
         return True
+    if isinstance(e, aiohttp.client_exceptions.ClientConnectorError):
+        return is_transient_error(e.os_error)
```

Comment:
Can you add the error message above? Also do you need to import aiohttp.client_exceptions?

Reply:
I don't need to import it because `import aiohttp` lets me refer to `aiohttp.client_exceptions`